terraform/azure/examples/aks/main.tf

```diff
@@ -0,0 +1,124 @@
+resource "azurerm_resource_group" "vnet" {
```

Comment:
I dont think this entire file should be in the examples? Shouldnt it be the main module?